EmotiBit.cpp

```diff
@@ -937,7 +959,7 @@ uint8_t EmotiBit::setup(String firmwareVariant)
 	led.setLED(uint8_t(EmotiBit::Led::BLUE), true);
 	led.send();
 	// ToDo: There is no catch right now for timeout. In case of timeout, EmotiBit still continues to complete setup() and proceed to update()
-	_emotiBitWiFi.begin(-1, 1);
+	_emotiBitWiFi.begin(-1, 1, 20000);
 	led.setLED(uint8_t(EmotiBit::Led::BLUE), false);
```

Comment:
The current behavior is to try and connect to a network in an infinite loop.
I think this change keep the behavior the same, i.e., an infinite connect loop in setup, AND also improves it by providing a software reset.
Also, agreed that it will help with enterprise WiFi too.

Reply:
Added this change

Reply:
On testing, i notice that if a user only adds 1 credential, and it happens to be the wrong one, then the Emotibit will restart every 40secs.

I am debating if this needs a documentation change, since the EmotiBit will be cycling through setup states.